guides/common/modules/proc_upgrading-hosts-to-next-major-release.adoc

```diff
@@ -41,15 +41,23 @@ ifndef::satellite[]
 # {foreman-installer} --enable-foreman-plugin-leapp
 ----
 endif::[]
-. If you are using a custom job template for the Leapp pre-upgrade check, configure the *leapp_preupgrade* remote execution feature to point to your template:
+. Optional: Customize the upgrade by modifying the default pre-upgrade check and upgrade job templates.
+.. Clone and modify the *Run preupgrade via Leapp* template, the *Run upgrade via Leapp* template, or both.
+For more information, see xref:customizing-job-templates[].
 .. In the {ProjectWebUI}, navigate to *Administer* > *Remote Execution Features*.
-.. Click *leapp_preupgrade*.
-.. In the *Job Template* dropdown menu, select your template.
+.. If you modified the pre-upgrade template, click *leapp_preupgrade*.
+If you modified the upgrade template, click *leapp_upgrade*.
+.. In the *Job Template* drop-down menu, select your template.
 .. Click *Submit*.
-. In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts*.
-. Select the hosts that you want to upgrade to the next major {EL} version.
-. From the *Schedule Remote Job* list, select *Preupgrade check with Leapp*.
-. When the check is finished, click the *Leapp preupgrade report* tab to see if Leapp has found any issues on your hosts.
+The *leap_preupgrade* and *leapp_upgrade* remote execution features now point to the modified templates.
+. Run a pre-upgrade check:
+.. In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts*.
+.. Select the hosts that you want to upgrade.
+.. If you want to upgrade to the latest available release, from the *Schedule a Job* drop-down menu, select *Preupgrade check with Leapp*.
+.. If you want to upgrade to a specific release version, click *Schedule a job*, and select *Leapp - Preupgrade* from the *Job category* list.
+Follow the job wizard to start the preupgrade check.
+. Review the results of the pre-upgrade check:
+.. When the check is finished, click the *Leapp preupgrade report* tab to see if Leapp has found any issues on your hosts.
```

Comment:
This currently does not exist in the new job invocation details page. Should we mention that until it is added there?